docs/release-notes/release-notes-0.19.0.md

```diff
@@ -44,6 +44,8 @@
 
 ## RPC Updates
 
+* [Allow arbitrary source public key for BuildRoute RPC](https://github.com/lightningnetwork/lnd/pull/9153) so that lnd's _*ChannelRouter*_ and _*routerrpc.Server*_ can build routes from the perspective of different nodes.
```

Comment:
wrap at 80 🙏 
(while we are here, we can fix the wrapping of the entry above (for 9017) as well)